### PR TITLE
Minecraft tutorial

### LINKED ISSUE #564: Does it run Minecraft?
Create a tutorial for Minecraft and Dockercraft.

### PR DESCRIPTION
**Pull request by everett-toews**
Closes #564

### CODE REVIEW

**Comment by lauracly** [comment]
Stephanie will do tech review

**Comment by stephamon** [comment]
Okay @everett-toews, aside from the one clarification suggestion above, everything worked perfectly! Although my stupid minecraft server dropped me in a river through a canyon in the middle of the night. Super awesome job, random world!

**Comment by rgbkrk** [comment]
Would you be able to add some screenshots or other imagery into this tutorial?

File: _tutorials/2015-12-10-minecraft.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+    $ docker create \
+      --name minecraft-data \
+      --volume /data \
+      itzg/minecraft-server
+    bf1e5afceadbc4731f3785947e99e42d2cd9af4f559af3602e8262834d4e4d1e
+    ```
+
+    The output of this `docker create` command is your application container ID.
+
+    **Note**: Take extreme care not to remove this container. If this container get's removed you will lose all of Minecraft world data.
```
Comment (rgbkrk) [inline _tutorials/2015-12-10-minecraft.md]: s/get's/gets/

s/lose all of Minecraft world data/lose all of your Minecraft world data/

**Comment by kmsholcomb** [comment]
HI @everett-toews. My suggested markup is in the attached Word doc. In addition to textual edits, I have some edits around the numbering in a few sections, and around body text that seems to have been formatted as code. I hope that the markup is clear. Please let me know if you have any questions. Thanks!

[Explore Minecraft on Carina_editsKH.docx](https://github.com/getcarina/getcarina.com/files/58587/Explore.Minecraft.on.Carina_editsKH.docx)

**Comment by everett-toews** [comment]
Thanks for the feedback everyone!

**Pull request merged**